server/pull_request.go

```diff
@@ -30,8 +30,10 @@ func (s *Server) handlePullRequestEvent(event *PullRequestEvent) {
 		s.addHacktoberfestLabel(pr)
 
 		if pr.RepoName == s.Config.EnterpriseTriggerReponame {
-			s.createEnterpriseTestsPendingStatus(pr)
-			go s.triggerEETestsforOrgMembers(pr)
+			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```

Comment:
Might be useful to refactor the `5 * time.Minute` as a constant somewhere else since this is being used 4 times.

Reply:
I think, I should fine tune this not to be such a constant value instead. Would it add a lot of value to be a constant?

Reply:
Do you foresee changing the timeout value dynamically during program execution ? That would be very surprising. We usually always default to a constant if we don't change a value.